docs/deployment.md

```diff
@@ -17,63 +17,59 @@ Inhalt:
 --->
 
 # {{ page.title }}
-In this article we will show you how to create a plug-in and install it in the Auto Importer. We will also show you how to set an installed plug-in as the import source in an import plan.
+{: .no_toc }
+So far, we used to run, debug and test plug-ins by loading them in *PiWeb Auto Importer* directly from their build output directory via command line parameter <span class="nowrap">`-pluginSearchPaths`</span>. While easy for testing and debugging, this is, of course, not a practicable way of distributing a plug-in after its release and even less so to deploy it in a production environment. In this article we will show you how to create an easily distributable plug-in package and how to install it in a *PiWeb Auto Importer* not running in development mode.
 
-## Create plug-in
-If the plug-in project uses our import sdk template as recommended by us. The build function `GeneratePluginPackageOnBuild` is automatically provided. This option is visible in the project file:\
-`<GeneratePluginPackageOnBuild>true</GeneratePluginPackageOnBuild>`\
-If the project is compiled, the necessary pip is automatically generated and placed inside your output directory.
+## Table of Contents
+{: .no_toc }
+1. TOC
+{:toc}
 
-## Install a plug-in
-The generated **pip** file can be installed directly in the Auto Importer as a plug-in.\
-This works via the install function in the Auto Importer, via drag & drop into the Auto Importer window and via file system.
+## Creating a plug-in package
+A *PiWeb Installable Plug-in* (.pip) package file is automatically created on every build of your plug-in as long as the *Import SDK* nuget is referenced by the project. The file is written to `$(PackageOutputPath)` which usually is (depending on your build configuration) either `bin\Debug` or `bin\Release`. A .pip file contains a the complete plug-in implementation and does not have any other dependencies. As a single file, it is easy to distribute.
 
-### Download compiled example plug-in
+{: .note}
+>You may switch building the package on and off by setting the `GeneratePluginPackageOnBuild` property in the project file:
+>```xml
+><GeneratePluginPackageOnBuild>true</GeneratePluginPackageOnBuild>
+>```
+
+## Distributing a plug-in package
+Unfortunately, there is currently no package management infrastructure available that could be used to manage, share and distribute packages centrally. You can, however, manage and distribute packages by your own means.
+
+## Installing a plug-in package
+Plug-in packages can directly be installed in *PiWeb Auto Importer* in two different ways:
+1. by dropping the package file to install anywhere in *PiWeb Auto Importer*, or
+1. by opening the plug-in management view via `File > Plug-ins...` and then clicking on `Install plug-in...` in the management actions dropdown. You will then be prompted to choose the package file to install.

```

Comment:
Here “1.” is used for the second time.

Reply:
That is actually intended. Jekyll takes care of the correct numbering when rendering. This makes rearranging easier.
![grafik](https://github.com/user-attachments/assets/b6eac5c0-370e-487a-901a-e042d0c5cfa4)